mitosheet/css/elements/SearchBar.css

```diff
@@ -0,0 +1,21 @@
+.mito-close-search-button {
+    border: none;
+    background: none;
+    display: flex;
+    align-items: center;
+    margin-left: 5px;
+}
+
+.mito-close-search-button:hover {
+    background-color: var(--mito-highlight-light);
+}
+
+.mito-search-bar {
+    position: absolute;
+    top: 72px;
+    right: 0;
+    background-color: var(--mito-background-highlight);
+    box-shadow: 1px 4px 6px var(--mito-light-gray);
```

Comment:
Is there a parametric color variable you can use for the shadow? E.g. other than -light-gray, which might change per theme?